doc/source/docs/user_guide/mapping.rst

```diff
@@ -257,16 +251,15 @@ the ``kind`` keyword argument in :meth:`~GeoDataFrame.plot`, and include:
 
 .. ipython:: python
 
-    gdf = world.head(10)
     @savefig pandas_line_plot.png
-    gdf.plot(kind='scatter', x="pop_est", y="gdp_md_est")
+    chicago.plot(kind="scatter", x="Pop2012", y="shape_area")
 
 You can also create these other plots using the ``GeoDataFrame.plot.<kind>`` accessor methods instead of providing the ``kind`` keyword argument.
 
 .. ipython:: python
 
-    @savefig pandas_bar_plot.png
-    gdf.plot.bar()
+    @savefig pandas_hist_plot.png
+    chicago[["Pop2012", "Pop2014", "geometry"]].plot.hist(alpha=.4)
```

Comment:
nitpicking, for this histogram to be readable, should it have an xaxis label (Community area Population) or a comment preceding the code to indicate what it shows. Iideally y axis would be integers as well as it's showing a frequency. 

Alternatively, could be the same as the old plot by just inlining the call to `.head()`.

Reply:
The old plot was not very useful, that is why I deliberately changed it. We just want to illustrate that you can still call pandas plot types, I wouldn't add other params to the plot here (labels...). But I'll add a comment.